web-ts/src/components/GeneralHeader.tsx

```diff
@@ -6,17 +6,14 @@ import { withTranslation, TFunction } from 'react-i18next'
 import buildConfig from '../constants/buildConfig'
 
 type PropsType = {
+  landingPath: string
```

Comment:
Not really. I first thought it may be better not to have dependencies to the `RoutePatterns` in components, but I think that is okay. Will use `RoutePatterns` here directly.